src/QsCompiler/Tests.Compiler/TypeCheckingTests.fs

```diff
@@ -88,6 +99,47 @@ module TypeCheckingTests =
         expect "LambdaInvalid6" [ Error ErrorCode.LocalVariableAlreadyExists ]
```

Comment:
Does the first line of ``supports lambda expressions`` need to be update now that there are more tests?

Reply:
Good idea, I updated the number.